views/default/simplesaml/settings/service_provider_advanced.php

```diff
@@ -50,6 +50,19 @@
 	$body .= elgg_format_element('div', [], $external_id);
 }
 
+$force_authentication = $plugin->getSetting('force_authentication');
+if ($force_authentication == $source_id) {
```

Comment:
Hmm, yeah.. You can see it as a issue, but I saw it as a feature :'P. Since you need to save it anyway before editing the settings.